src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/custom.py

```diff
@@ -8,10 +8,12 @@
 from __future__ import print_function
 from sys import stderr
 
+from azure.storage.blob import BlockBlobService#, AppendBlobService, PageBlobService
```

Comment:
Should you remove the commented imports?

---
In reply to: [74279181](https://github.com/Azure/azure-cli/pull/673#discussion_r74279181) [](ancestors = 74279181)